jobs/config.json

```diff
@@ -2539,7 +2539,7 @@
       "--gcp-zone=us-central1-b",
       "--mode=docker",
       "--provider=gce",
-      "--test_args=--ginkgo.focus=\\[Feature:Empty\\] --allowed-not-ready-nodes=10 --node-schedulable-timeout=60m --cluster-ip-range=10.160.0.0/13",
```

Comment:
Shouldn't we delete the --cluster-ip-range flag?

Reply:
Wanted to let it still stay in order to allow passing custom ranges.